library/seq.lisp

```diff
@@ -562,6 +562,12 @@ It attempts to rebalance with a minimum of array copying."
                           (cl:rest leaf-arrays)
                           :initial-value `(LeafArray ,(cl:funcall make-ary (cl:first leaf-arrays))))))))))
 
+(cl:defmethod cl:print-object :around ((self seq) stream)
```

Comment:
can you write a brief comment describing that this is a bit of a hack because of blah blah blah... (why :around, why not a directly on seq, etc)